### PR TITLE
Add Support to 'SCRIPT KILL' Command

### PR DESCRIPTION
**Pull request by shacharPash**
link to the command: https://redis.io/commands/script-kill/
part of #2055

### CODE REVIEW

File: src/StackExchange.Redis/Interfaces/IServer.cs [hunk 1]
```diff
@@ -448,6 +448,17 @@ public partial interface IServer : IRedis
         /// <inheritdoc cref="ScriptLoad(LuaScript, CommandFlags)"/>
         Task<LoadedLuaScript> ScriptLoadAsync(LuaScript script, CommandFlags flags = CommandFlags.None);
 
+         /// <summary>
+        /// Kills the currently executing EVAL script.
+        /// </summary>
+        /// <param name="flags">The command flags to use.</param>
+        /// <returns>Simple string reply.</returns>
+        /// <remarks><seealso href="https://redis.io/commands/script-kill/"/></remarks>
+        void ScriptKill(CommandFlags flags = CommandFlags.None);
```
[the remark this answers is not in the document]
Comment (shacharPash) [reply src/StackExchange.Redis/Interfaces/IServer.cs:456]: I deleted the <returns> line from the documentation, I looked at the similar command: 'ScriptFlush' and there it was done this way.

[the remark this answers is not in the document]
Comment (NickCraver) [reply src/StackExchange.Redis/Interfaces/IServer.cs:456]: @shacharPash Sorry should have been clearer: I meant in the Redis server documentation, it returns a string so we should return it here :)

**Comment by mgravell** [comment]
I have two concerns over SCRIPT KILL

1. That is won't actually work in our scenario: new connections most likely
won't be able to connect in the face of a blocking script, due to the
handshake operations we have - and existing connections are likely to be
stuck behind other operations, maybe just heartbeats
2. That it is a command that should almost never be used

Is this a "we have a problem where we need this" thing, or is it a "here's
a mm issuing command" thing? If the latter, I wonder whether on this
particular occasion: it might be better to stay missing

**Comment by NickCraver** [comment]
Since this doesn't make sense on a multiplexed connection, we're going to close this on out - it'd just almost never do what users expect. We can revisit this with connection pooling but it doesn't make sense the way connections work today.

**Pull request closed without merging**